buildSrc/src/main/java/dependencies/Versions.kt

```diff
@@ -5,12 +5,13 @@ object Versions {
     const val android_min_sdk = 21
     const val android_target_sdk = 31
 
-    const val androidx_fragment = "1.4.1"
+    const val androidx_fragment = "1.5.3"
```

Comment:
We need this version as it adds the `OnBackPressedDispatcherOwner` interface to `ComponentDialog`.